src/dns.c

```diff
@@ -1359,7 +1376,8 @@ hsk_dns_rd_write(
     case HSK_DNS_DNAME: {
       hsk_dns_dname_rd_t *r = (hsk_dns_dname_rd_t *)rd;
 
-      size += hsk_dns_name_write(r->target, data, cmp);
+      // no label compression
```

Comment:
So DNS_DNAME, DNS_SRV, DNS_RP, DNS_RRSIG and DNS_NSEC no longer use label compression?

Reply:
Correct! explanation in commit message: 5d846f9cdb1fe3425b34e8f32472a38e68ab80e8